frontend/src/components/ZetaneDrawflowEditor.js

```diff
@@ -176,119 +176,110 @@ export default class Drawflow {
         this.ele_selected = e.target.closest(".drawflow-node");
       }
     }
-    switch (this.ele_selected.classList[0]) {
```

Comment:
this may be a question for @FGRCL but why does the linter want if / else if / else instead of a `switch`? i don't think i agree that should be an enforced code standard (to just remove case statements?)

Reply:
I'm answering this on my phone so I would need to check to be sure. I think it was complaining about a specific operation that was happening in the switch statement. So it's not a specific rule about prohibiting switch statements. I'll find the specific rule